tests/tests_integration/test_api/test_simulators/test_runs.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from cognite.client._cognite_client import CogniteClient
+from cognite.client.data_classes.simulators.runs import SimulationRunWrite
+
+
+@pytest.mark.usefixtures("seed_resource_names", "seed_simulator_routine_revisions")
+class TestSimulatorRuns:
+    def test_create_run(
+        self, cognite_client: CogniteClient, seed_simulator_routine_revisions, seed_resource_names
+    ) -> None:
+        routine_external_id = seed_resource_names["simulator_routine_external_id"]
+        created_runs = cognite_client.simulators.runs.create(
+            [
+                SimulationRunWrite(
+                    run_type="external",
+                    routine_external_id=routine_external_id,
+                )
+            ]
+        )
+        assert created_runs[0].routine_external_id == routine_external_id
+        assert created_runs[0].id is not None
+
+    def test_list_filtering(self, cognite_client: CogniteClient, seed_resource_names) -> None:
+        routine_external_id = seed_resource_names["simulator_routine_external_id"]
+        status = ["running", "success", "failure"]
+        for _ in range(3):
+            created_runs = cognite_client.simulators.runs.create(
+                [
+                    SimulationRunWrite(
+                        run_type="external",
+                        routine_external_id=routine_external_id,
+                    )
+                ]
```

Comment:
with `self._CREATE_LIMIT = 1` you should be able to create three with just one function call

```suggestion
                [
                    SimulationRunWrite(
                        run_type="external",
                        routine_external_id=routine_external_id,
                    )
                ] for _ in statuses
```